### PR TITLE
Whitespace

### PR DESCRIPTION
**Pull request by Arch-Shaman**
Double space whitespace is evil.
Banish it from my sanctuary.

### CODE REVIEW

**Comment by GoogleFrog** [comment]
I don't see cawidgets here.

**Comment by Arch-Shaman** [comment]
Accidentally pushed it to `patch-3`. Fixed.

File: LuaUI/Utilities/json.lua [hunk 1]
```diff
@@ -224,18 +224,16 @@ end
...
-    endPos = endPos + 1
-  end
-  local stringValue = 'return ' .. string.sub(s,startPos, endPos-1)
-  local stringEval = loadstring(stringValue)
-  assert(stringEval, 'Failed to scan number [ ' .. stringValue .. '] in JSON string at position ' .. startPos .. ' : ' .. endPos)
-  return stringEval(), endPos
+	local endPos = startPos+1
+	local stringLen = string.len(s)
+	local acceptableChars = "+-0123456789.e"
+	while (string.find(acceptableChars, string.sub(s,endPos,endPos), 1, true) and endPos<=stringLen) do
```
Comment (GoogleFrog) [inline LuaUI/Utilities/json.lua:230]: If you want to put a bit more work in, you could go through and fix the commas and comparisons.  A comma should always be followed by a space or a newline. The ==, <= etc.. signs should have spaces on each side.

**Comment by GoogleFrog** [comment]
I've been writing this part of the wiki https://zero-k.info/mediawiki/index.php?title=Zero-K:Developing#Style_Guide

File: LuaUI/Utilities/json.lua (hunk 1 above)
Comment (Arch-Shaman) [reply LuaUI/Utilities/json.lua:230]: Sure I'll put in a bit of extra work to bring it up to compliance.

**Comment by Arch-Shaman** [comment]
This is On-Hold for a day due to me being out of commission today.

File: LuaUI/Utilities/json.lua (hunk 1 above)
Comment (Arch-Shaman) [reply LuaUI/Utilities/json.lua]: This evil has been banished.

**Pull request merged**